src/main/webapp/app/home/home.component.html

```diff
@@ -1,44 +1,41 @@
-<div class="row">
-    <div class="col-md-3">
-        <span class="hipster img-fluid rounded"></span>
-    </div>
-
-    <div class="col-md-9">
-        <h1 class="display-4" jhiTranslate="home.title">Welcome, Java Hipster!</h1>
-
-        <p class="lead" jhiTranslate="home.subtitle">This is your homepage</p>
-
-        <div [ngSwitch]="isAuthenticated()">
-            <div class="alert alert-success" *ngSwitchCase="true">
-                <span id="home-logged-message" *ngIf="account" jhiTranslate="home.logged.message"
-                    [translateValues]="{ username: account.login }">You are logged in as user "{{ account.login }}".</span>
+<div [ngSwitch]="isAuthenticated()">
+    <div class="row justify-content-center ">
+        <div class="col-12 ">
+            <div class="entityWrapper home">
+                <h8 class="text-primary font-weight-bold" jhiTranslate="home.title">Welcome</h8>
+                <h8 class="text-primary" id="home-logged-message" *ngIf="account" jhiTranslate="home.title2"
+                    [translateValues]="{ username: account.login }">
+                    <span class="title-color"> {{ account.login }}!.</span></h8>
             </div>
-
-            <div class="alert alert-warning" *ngSwitchCase="false">
-                <span jhiTranslate="global.messages.info.authenticated.prefix">If you want to </span>
-                <a class="alert-link" (click)="login()" jhiTranslate="global.messages.info.authenticated.link">sign in</a><span jhiTranslate="global.messages.info.authenticated.suffix">, you can try the default accounts:<br/>- Administrator (login="admin" and password="admin") <br/>- Curator (login="user" and password="user") <br/>- Nurse (login="viewer" and password="viewer").</span>
+            <div class="mainWrapper entityInfoWrapper">
+                <p class="text-secondary p " jhiTranslate="home.navigate">Navigate</p>
             </div>
-
-            <div class="alert alert-warning" *ngSwitchCase="false">
-                <span jhiTranslate="global.messages.info.register.noaccount">You don't have an account yet?</span>&nbsp;
-                <a class="alert-link" routerLink="account/register" jhiTranslate="global.messages.info.register.link">Register a new account</a>
+            <div class="sectionWrapper">
+                <a class="box1" routerLink="/therapeutic-regime">
+                    <div class="img">
+                        <fa-icon icon="pills" size="5x" style="color:lightgray;"></fa-icon>
+                    </div>
+                    <div class="img">
+                        <span jhiTranslate="home.link.therapeutic-regime" style="font-size:0.79rem !important">Therapeutical Regime</span>
```

Comment:
Can't the styles added on lines 16, 24, 27, 32 and 34 be added to the `home.scss` file?

Reply:
done